src/core/core.scale.js

```diff
@@ -1292,17 +1300,19 @@ export default class Scale extends Element {
       }
 
       items.push({
-        rotation,
         label,
         font,
-        color,
-        strokeColor,
-        strokeWidth,
         textOffset,
-        textAlign: tickTextAlign,
-        textBaseline,
-        translation: [x, y],
-        backdrop,
+        renderTextOptions: {
```

Comment:
The seems badly named as its referring to some method used to render the text and thus exposes (and freezes) internal logic.
I think plain `options` would work. Also the `textOffset` should go in there too.

IMO we should make the ticks and labels to be (animated) elements in future releases, that is why I suggest plain `options`. We could just return the elements as labelItems then.

Reply:
I don't disagree - obviously the renderTextOptions name came about because it stores a RenderTextOpts typed object. Why textOffset and, for that matter, font are passed separately into renderText() rather than in the options collection is also a mystery that I didn't dive into. I certainly agree that it seems to make sense to fold font and textOffset into the RenderTextOpts object, which simplifies the renderText signature.

Reply:
@kurkle, renamed to just options, didn't make any changes to `textOffset` because I didn't want this change to break any implementations that relied on renderText(). However, now lint tests are failing (passing locally) - seems like perhaps an intermittent issue? Can checks be retriggered?